cylc/flow/network/scan.py

```diff
@@ -164,8 +165,10 @@ def scan_many(items, methods=None, timeout=None, ordered=False):
         try:
             msg = await client.async_request(method)
         except ClientTimeout as exc:
+            LOG.exception(f"ClientTimeout-reg:{reg}|host:{host}|port:{port}")
```

Comment:
```suggestion
            LOG.exception(f"ClientTimeout: name:{reg}, host:{host}, port:{port}")
```

A minor point, but we don't use `|` as a separator in message strings anywhere else. Otherwise all good.

Reply:
quick question: did you check if that won't exceed the 80 characters limit?

Reply:
I did not check, and you're right it does go over 80!  (I wonder if I can redo the suggestion...)